jerry-core/debugger/debugger-ws.h

```diff
@@ -85,7 +102,7 @@ bool jerry_debugger_accept_connection (void);
 void jerry_debugger_close_connection (void);
 
 bool jerry_debugger_send (size_t data_size);
-bool jerry_debugger_receive (void);
+bool jerry_debugger_receive (jerry_debugger_string_data_t **message_data_p);
```

Comment:
Hm, perhaps `jerry_debugger_uint8_data_t` would be better, because in the future we might receive non-string data, although it will still be a byte data. And `jerry_debugger_string_data_t` -> `jerry_debugger_uint8_data_t`